.github/workflows/trigger.yml

```diff
@@ -0,0 +1,19 @@
+# Copyright 2021, Proofcraft Pty Ltd
+#
+# SPDX-License-Identifier: BSD-2-Clause
+
+# Trigger repository dispatch on main test repos
+name: Trigger
+
+on:
+  push:
+    branches: [master]
+
+jobs:
+  trigger:
+    name: Repository Dispatch
+    runs-on: ubuntu-latest
+    steps:
+    - uses: seL4/ci-actions/trigger@master
+      with:
+        token: ${{ secrets.PRIV_REPO_TOKEN }}
```

Comment:
Since we don not have these on github forks, can you add a check that this runs only on the "seL4" organization. That avoids getting failure reports everytime master is updated in the fork (which happens when testing PR on forks using the github CI)

Reply:
This commit is already in master, but I agree that we should adjust that. Could you make an issue for it?

Reply:
Done, https://github.com/seL4/projects_libs/issues/16